app/controllers/happy_things_controller.rb

```diff
@@ -35,7 +35,7 @@ def destroy
 
   def analytics
     @happy_count = HappyThing.where(user: current_user).size
-    @words_for_wordcloud = WordAggregator.get_aggregated_words(current_user, 50)
+    @words_for_wordcloud = WordAggregator.get_aggregated_words(current_user, 40)
```

Comment:
What makes you decide between 40 or 50? Should this be a constant in the `WordAggregator` or a method that explains your judgement ?